src/openms/source/SIMULATION/RawTandemMSSignalSimulation.cpp

```diff
@@ -180,11 +180,11 @@ namespace OpenMS
         precomputed_MS2[i_f].push_back(p);
       }
 
-      precomputed_MS2[i_f].setMSLevel(2);
+      /*precomputed_MS2[i_f].setMSLevel(2);
```

Comment:
omitting this does not work: the simulated spectrum is in `tmp_spec` and only the peaks are copied.
If you want to optimize, this will probably do it:
```
      Int prec_charge = features[i_f].getCharge();

      if (tandem_mode && svm_model_charges.count(prec_charge))
      {
        svm_spec_gen_set.simulate(precomputed_MS2[i_f], seq, rnd_gen_->getBiologicalRng(), prec_charge);
        precomputed_MS2[i_f].setMSLevel(2);
        Precursor prec;
        prec.setMZ(features[i_f].getMZ());
        precomputed_MS2[i_f].setPrecursors(std::vector<Precursor>(1, prec));
      }
      else
      {
        simple_generator.getSpectrum(precomputed_MS2[i_f], seq, 1, prec_charge, prec_charge);
      }

      precomputed_MS2[i_f].setMetaValue("FeatureID", static_cast<String>(features[i_f].getUniqueId()));
```
But you might as well undo the changes.